release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Separate downloader for istioctl
+#
+# This file will be fetched as: curl -L https://git.io/downloadIstioCtl | sh -
+#
+detect_os() {
+  OS=$(uname)
+  case $OS in
+    'Linux')
+      OSEXT='linux'
+      FILENAME='istioctl'
+      ;;
+    'WindowsNT')
+      OSEXT='win.exe'
+      FILENAME='istioctl.exe'
+      ;;
+    'Darwin') 
+      OSEXT='osx'
+      FILENAME='istioctl'
+      ;;
+    *) ;;
+  esac
+}
+
+get_version() {
+  if [ "x${ISTIO_VERSION}" = "x" ] ; then
```

Comment:
why x?

Reply:
@howardjohn you know.... you've asked a question that I _shouldv'e_ asked when I saw this same line of code here:

https://github.com/istio/istio/blob/8c94bcc91c2e8591da4059660619878ce2725ae2/release/downloadIstioCandidate.sh#L24

At first, I was thinking that this was some nifty way of checking to see if $ISTIO_VERSION was set, but you don't have to do any fancy to find that out.... not sure, I used the same condition in my script, unaware of the reason, TBH...

Reply:
@howardjohn I think you've convinced me that it's not needed.... unless someone else can shed some light here...

Reply:
Probably because it's using single `[` instead of double `[[`, and with single an empty variable will blow up. So yes this is a "nifty way of checking to see if $ISTIO_VERSION was set".

Reply:
@dwradcliffe thanks for the info... and just in time, b/c I was about to amend this PR. :-)

Reply:
:+1: For compatibility probably best to leave it even though it's ugly.